ROADMAP.md

```diff
@@ -33,6 +33,12 @@ In the future, we want to allow third party tools to be plugged into Docker Swar
 of much bigger clusters.
 
 The work has started on defining the "cluster" interface in [#393](https://github.com/docker/swarm/issues/393)
-qThe first integration will be Mesos but other tools will arrive after, like kubernetes.
+
+The first integration will be Mesos but other tools will arrive after, like kubernetes.
 
 * [ ] Cluster Drivers
+
+Project Planning
+================
+* [0.2.0 Milestone](https://github.com/docker/swarm/wiki/0.2.0-Milestone-Project-Page)
```

Comment:
How about just replicating the chart on the wiki home page, which maps the Platform Milestones to the Swarm Milestones, and clearly defines the Project Page:

| Platform Version | Swarm Milestone | Project Page | 
|------------------|-----------------|--------------|
| Docker 1.6.0 | [0.2.0](https://github.com/docker/swarm/milestones/0.2.0) | [Project Page](https://github.com/docker/swarm/wiki/0.2.0-Milestone-Project-Page) | 
| Docker 1.7.0 | [1.0.0](https://github.com/docker/swarm/milestones/1.0.0) | [Project Page](https://github.com/docker/swarm/wiki/1.0.0-Milestone-Project-Page) | 

I omitted the date column: this is something that could get out of sync.